pyproject.toml

```diff
@@ -2,7 +2,7 @@
 line-length = 100
 skip-string-normalization = true
 target-version = ["py38"]
-exclude='\.eggs|\.git|\.mypy_cache|\.tox|\.venv|_build|buck-out|build|dist'
+exclude='\.eggs|\.git|\.mypy_cache|\.tox|\.venv|_build|buck-out|build|dist|web'
```

Comment:
Nice. This will conflict with a different fix I have in the remove-auth branch, but yours is better. I'll remove the fix from my branch and rely on this one instead.

(A note: we both did a bad thing by including this fix in our feature branches; better would be to cherry pick this out to a PR against main so that everyone could have benefited from our genius independently of these big feature branches we're needing to review before merge.)